eng/pipelines/templates/steps/analyze.yml

```diff
@@ -26,6 +26,17 @@ steps:
       python find_invalid_versions.py --always-succeed --service=${{parameters.ServiceDirectory}}
     displayName: Find Invalid Versions
 
+  - pwsh: |
+      Get-ChildItem .\ -Filter "*.py" |
+      Foreach-Object {
+        (Get-Content $_ -Raw) -match "\r\n$" && Write-Output $_
```

Comment:
Does this work? It looks like it might only check the end of the file for crlf, as I believe Raw will cause you to return the full file contents. If you want to check for it everywhere I think you want to remove the `$` from the regex.